src/utils/fxa.ts

```diff
@@ -349,6 +349,61 @@ async function applyCoupon(
 }
 /* c8 ignore stop */
 
+/**
+ * @see https://mozilla.github.io/ecosystem-platform/api#tag/Devices-and-Sessions/operation/getAccountAttached_clients
+ */
+export type FxaGetAccountAttachedClients = {
+  clientId: string;
+  deviceId: number;
+  sessionTokenId: string;
+  refreshTokenId: string;
+  isCurrentSession: boolean;
+  deviceType: string;
+  name: string;
+  createdTime: string;
+  lastAccessTime: string;
+  scope: string[];
+  userAgent: string;
+  createdTimeFormatted?: string;
+  approximateLastAccessTime?: number;
+  location?: {
+    city: string;
+    country: string;
+    state: string;
+    stateCode: string;
+  };
+  os?: string;
+};
+
+// Not covered by tests; mostly side-effects. See test-coverage.md#mock-heavy
+/* c8 ignore start */
+async function getAttachedClients(
+  bearerToken: string,
+): Promise<FxaGetAccountAttachedClients[]> {
+  const endpointUrl = `${envVars.OAUTH_ACCOUNT_URI}/account/attached_clients`;
+  try {
+    const response = await fetch(endpointUrl, {
+      headers: {
+        Accept: "application/json",
+        Authorization: `Bearer ${bearerToken}`,
+      },
+    });
+    const responseJson = await response.json();
+    if (!response.ok) throw new Error(responseJson);
```

Comment:
I think we need to JSON.stringfy this

Reply:
Yup! Added in [db83e13](https://github.com/mozilla/blurts-server/pull/5261/commits/db83e13fdc226596dc2646b2c363c27f78557ca3).